Assets/AltTester/Editor/AltTesterEditorWindow.cs

```diff
@@ -214,19 +214,21 @@ public static void CheckDesktopVersionRequest()
                     {
                         regex = new System.Text.RegularExpressions.Regex(@"https://altom.com/app/uploads/AltTester/desktop/AltTesterDesktop[\w\.]*.dmg");
```

Comment:
this suggestion is not related to the bug, but it's a nice change, if you ask me :)
use a variable for the builds location, and use that variable here. if in the future we'll change to a new domain, it'll be easier to change it once.